dataland-internal-storage/src/main/kotlin/org/dataland/datalandinternalstorage/services/ConfigurationBackendControllerApi.kt

```diff
@@ -0,0 +1,24 @@
+package org.dataland.datalandinternalstorage.services
+
+import org.dataland.datalandbackend.openApiClient.api.NonPersistedDataControllerApi
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+/**
+ * This class holds the configuration properties and the beans to auto-configure this library in a spring-boot
+ * environment
+ */
+@Configuration
+class ConfigurationBackendControllerApi(
```

Comment:
Shouldn't this be a "ConfigurationNonPersitedDataControllerApi" or - if you prefer - a `ConfugrationBackendClientApi`?

Reply:
Also renamed

Reply:
ok